astroquery/gaia/__init__.py

```diff
@@ -38,4 +39,6 @@ class Conf(_config.ConfigNamespace):
 
 from .core import Gaia, GaiaClass
 
+"""
+
 __all__ = ['Gaia', 'GaiaClass', 'Conf', 'conf']
```

Comment:
but `Conf` is still used here, which is why the docs are failing

Reply:
Yes, you are right.